### PR TITLE
Conf Changes

### PR DESCRIPTION
**Pull request by mumutyal**

### CODE REVIEW

File: block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf [hunk 1]
```diff
@@ -46,7 +46,7 @@ GOPATH=$GOPATH  					# Read from environment
 # Helm chart install
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf]: The vars such as `PVG_BX_DASH_S`, `PVG_CLUSTER_CRUISER`, `BLOCK_PLUGIN_HOME` and 
```
IMAGE_REGISTRY=registry.stage1.ng.bluemix.net
USER_NAMESPACE=contsto2
PLUGIN_IMAGE=ibmcloud-block-storage-attacher
PLUGIN_BUILD=$PLUGIN_BUILD
```
are not updated in this jp-tok.conf file

File: block-storage-attacher/tests/e2e/conf/prod-us-south.conf [hunk 2]
```diff
@@ -3,7 +3,7 @@ PVG_BX_USER=contsto2@in.ibm.com
 PVG_BX_PWD=$PVG_BX_PWD			# Read from environment
 PVG_BX_DASH_C=e242f140687cd68a8e037b26680e0f04
 PVG_BX_DASH_O=contsto2@in.ibm.com
-PVG_BX_DASH_S=ibmc-block-e2e-test
+PVG_BX_DASH_S=ibmc-blockattach-volume--e2e-test
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-south.conf]: This value is incorrect.

File: block-storage-attacher/tests/e2e/conf/prod-us-south.conf [hunk 3]
```diff
@@ -13,7 +13,7 @@ PVG_SL_USERNAME=1186049_contdep@us.ibm.com
 PVG_SL_API_KEY=$PVG_SL_API_KEY	# Read from environment
 
 # Armada cluster create params
-PVG_CLUSTER_CRUISER=ibmc-block-e2e-test
+PVG_CLUSTER_CRUISER=ibmc-blockattach-volume--e2e-test
```
Comment (akgunjal) [inline block-storage-attacher/tests/e2e/conf/prod-us-south.conf]: This value is incorrect.

File: block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf (hunk 1 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/prod-jp-tok.conf]: In this file those lines were commented . Un commented and updated

File: block-storage-attacher/tests/e2e/conf/prod-us-south.conf (hunk 2 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/prod-us-south.conf]: done

File: block-storage-attacher/tests/e2e/conf/prod-us-south.conf (hunk 3 above)
Comment (mumutyal) [reply block-storage-attacher/tests/e2e/conf/prod-us-south.conf]: done

**Pull request merged**